lib/prop/leaky_bucket_strategy.rb

```diff
@@ -5,40 +5,73 @@
 module Prop
   class LeakyBucketStrategy
     class << self
+      def _throttle_leaky_bucket(handle, key, cache_key, options)
+        (over_limit, bucket) = options.key?(:decrement) ?
+          decrement(cache_key, options.fetch(:decrement), options) :
+          increment(cache_key, options.fetch(:increment, 1), options)
+
+        if over_limit
+          [true, bucket]
+        else
+          [false, bucket]
+        end
```

Comment:
Could be simplified to. `[over_limit, bucket]`

Reply:
done